data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>It might be a little clearer if you just have a bulleted list of just the four column names and the explanations in a separate sentence. "Starttime and stoptime will be used for determining how long a trip was. The station_ids will be used to determine how many trips started and/or stopped at a particular station." Maybe?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Hmm, I kind of disagree - I do things like this in devsite docs, but I'm also happy to have a TW overrule us and be the final word </p>

Reply:
<p>I don't feel strongly enough about this one way or another. If a TW has previously ok'd something similar, that's fine with me </p>

Reply:
<p>Let's let the final TW reviewer decide :) </p>